Classes/WPEditorToolbarButton.m

```diff
@@ -200,6 +205,16 @@ - (void)setNormalTintColor:(UIColor *)normalTintColor
 	}
 }
 
+- (void)setDisabledTintColor:(UIColor *)disabledTintColor
+{
+    if (_disabledTintColor != disabledTintColor) {
```

Comment:
Purely a matter of preference, but maybe check for equality and return and avoid nesting, vs checking for inequality and doing the work inside an if block.